public/js/actions/ArticleActions.js

```diff
@@ -1,13 +1,34 @@
 import dispatcher from '../dispatcher.js'
 
 export default class ArticleActions {
-    // TODO: Have different fetch calls for top/hot articles
-    static fetchArticles() {
+
+    static fetchArticles(sort, lastArticle) {
```

Comment:
If you guys can think of a better way to design the action for fetching the newest/top let me know